circuits/offchain/credentialAtomicQueryV3OffChain.circom

```diff
@@ -182,7 +197,50 @@ template credentialAtomicQueryV3OffChain(issuerLevels, claimLevels, valueArraySi
 
     querySatisfied === 1;
 
-    /* ProfileID calculation */
+    /////////////////////////////////////////////////////////////////
+    // Modifier/Computation Operators Processing
+    /////////////////////////////////////////////////////////////////
+
+    // selective disclosure calculation
+    // no need to calc anything, fieldValue is just passed as an output
+
+    // nullifier calculation
+    signal isNullifyOp <== IsEqual()([operator, 17]);
+    signal nullifier <== Nullify()(
+        isNullifyOp,
+        userGenesisID,
+        claimSubjectProfileNonce,
+        fieldValue,
+        value[0] // get csr from value array
+    );
+
+    /////////////////////////////////////////////////////////////////
+    // Operator Output Preparation
+    /////////////////////////////////////////////////////////////////
+
+    // parse operator to bits
+    signal opBits[5] <== Num2Bits(5)(operator); // values 0-15 are query operators, 16-31 - modifiers/computations
+
+    // output value calculation
+    signal modifierOutput <== Mux4()(
+        s <== [opBits[0], opBits[1], opBits[2], opBits[3]],
+
+        c <== [
+            fieldValue, // 16 - selective disclosure (16-16 = index 0)
+            nullifier, // 17 - nullify (17-16 = index 1)
+            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 // 18-31 - not used
+        ]
+    );
+
+    // output value only if modifier operation was selected
+    operatorOutput <== Mux1()(
+        c <== [0, modifierOutput], // output 0 for non-modifier operations
```

Comment:
should ib be 0 or 1 for non-modifier operations

Reply:
My idea was to return 0 for verifier libs to not even think of it's verification for regular query operators.
But could be 1 with verification in auth libs and zkp verifier. Both would work.